.github/workflows/release.yaml

```diff
@@ -20,16 +20,26 @@ jobs:
         with:
           go-version: '>=1.17.0'
       - run: go install github.com/goreleaser/nfpm/v2/cmd/nfpm@latest
-        name: Install dependencies
-
+        name: Install dependencies      
+      - name: Install node.js dependencies
+        run: | 
+              cd ./ui
+              npm install npm -g 
+              npm ci --no-optional 
+              cd -
```

Comment:
I'm pretty sure you can delete the `cd -` as it is not really doing anything (unless I'm missing something)

```suggestion
```